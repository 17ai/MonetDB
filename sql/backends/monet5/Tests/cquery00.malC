# some simple MAL tests

cquery.dump();

cquery.show("unknown","query");

(tick:bat[:timestamp],mod:bat[:str],fcn:bat[:str], status:bat[:str], time:bat[:lng],error:bat[:str]) := cquery.log();

io.print(tick,mod,fcn,status,time,error);
