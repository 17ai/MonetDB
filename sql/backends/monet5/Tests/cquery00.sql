-- test ordinary procedure call to update a stream
create stream table testing (a int);
insert into testing values(123);

create table results (a int);

create procedure myproc()
begin 
	insert into results select a from sys.testing; 
END;

-- a continuous procedure can be called like any other procedure
call myproc();

select * from results;

select * from functions where name = 'myproc';

drop procedure myproc;
drop table results;
drop table testing;
