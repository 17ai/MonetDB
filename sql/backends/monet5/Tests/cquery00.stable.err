stderr of test 'cquery00` in directory 'sql/backends/monet5` itself:


# 22:29:13 >  
# 22:29:13 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=36097" "--set" "mapi_usock=/var/tmp/mtest-16121/.s.monetdb.36097" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/mk/cquery//Linux/var/MonetDB/mTests_sql_backends_monet5"
# 22:29:13 >  

# builtin opt 	gdk_dbpath = /export/scratch1/mk/cquery//Linux/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 36097
# cmdline opt 	mapi_usock = /var/tmp/mtest-16121/.s.monetdb.36097
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /export/scratch1/mk/cquery//Linux/var/MonetDB/mTests_sql_backends_monet5
# cmdline opt 	gdk_debug = 536870922
#scheduler status init

# 22:29:14 >  
# 22:29:14 >  "mclient" "-lmal" "-ftest" "-Eutf-8" "--host=/var/tmp/mtest-16121" "--port=36097"
# 22:29:14 >  

MAPI  = (monetdb) /var/tmp/mtest-16121/.s.monetdb.36097
QUERY = # some simple MAL tests
        
        cquery.dump();
        
        cquery.show("unknown","query");
        
        (tick:bat[:timestamp],mod:bat[:str],fcn:bat[:str], time:bat[:lng],error:bat[:str]) := cquery.log();
        
        io.print(tick,mod,fcn,time,error);






# 22:29:14 >  
# 22:29:14 >  "Done."
# 22:29:14 >  

