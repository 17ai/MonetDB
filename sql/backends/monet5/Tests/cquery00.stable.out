stdout of test 'cquery00` in directory 'sql/backends/monet5` itself:


# 22:29:13 >  
# 22:29:13 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=36097" "--set" "mapi_usock=/var/tmp/mtest-16121/.s.monetdb.36097" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/mk/cquery//Linux/var/MonetDB/mTests_sql_backends_monet5"
# 22:29:13 >  

# MonetDB 5 server v11.26.0
# This is an unreleased version
# Serving database 'mTests_sql_backends_monet5', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.589 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.da.cwi.nl:36097/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-16121/.s.monetdb.36097
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 22:29:14 >  
# 22:29:14 >  "mclient" "-lmal" "-ftest" "-Eutf-8" "--host=/var/tmp/mtest-16121" "--port=36097"
# 22:29:14 >  


# 22:29:14 >  
# 22:29:14 >  "Done."
# 22:29:14 >  

